pkg/providers/vsphere/vmprovider.go

```diff
@@ -44,10 +43,6 @@ const (
 
 	// taskHistoryCollectorPageSize represents the max count to read from task manager in one iteration.
 	taskHistoryCollectorPageSize = 10
-
-	ovfCacheMaxItem                 = 100
-	ovfCacheItemExpiration          = 30 * time.Minute
-	ovfCacheExpirationCheckInterval = 5 * time.Minute
```

Comment:
Closest place GitHub allows me to comment: the `type VersionedOVFEnvelope struct ...` block below can also be deleted from this file.